domain/src/main/kotlin/no/nav/su/se/bakover/domain/søknadsbehandling/Statusovergang.kt

```diff
@@ -205,30 +206,31 @@ abstract class Statusovergang<L, T> : StatusovergangVisitor {
 
         private fun saksbehandlerOgAttestantErForskjellig(
             søknadsbehandling: Søknadsbehandling.TilAttestering,
-            attestering: Attestering
+            attestering: Attestering,
         ): Boolean = søknadsbehandling.saksbehandler.navIdent != attestering.attestant.navIdent
     }
 
     class OppdaterStønadsperiode(
         private val oppdatertStønadsperiode: Stønadsperiode,
-        private val tidligereSøknadsbehandlinger: List<Søknadsbehandling>
+        private val sak: Sak,
     ) : Statusovergang<OppdaterStønadsperiode.KunneIkkeOppdatereStønadsperiode, Søknadsbehandling.Vilkårsvurdert>() {
 
         sealed class KunneIkkeOppdatereStønadsperiode {
             data class KunneIkkeOppdatereGrunnlagsdata(val feil: KunneIkkeLageGrunnlagsdata) :
                 KunneIkkeOppdatereStønadsperiode()
 
-            object StønadsperiodeOverlapperMedEksisterendeSøknadsbehandling :
-                KunneIkkeOppdatereStønadsperiode()
+            object StønadsperiodeOverlapperMedEksisterendeStønadsperiode : KunneIkkeOppdatereStønadsperiode()
```

Comment:
```suggestion
            object StønadsperiodeOverlapperMedEksisterendeIkkeOpphørtStønadsperiode : KunneIkkeOppdatereStønadsperiode()
```
I påvente av innspill fra fag :shrug: Gjetter på at de ikke kaller historiske stønadsperioder for aktive.